src/components/TablePeople.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { Person } from '../types';
+import { InfoPerson } from './Loader/InfoPerson';
+
+type Prors = {
```

Comment:
The type name 'Prors' appears to be a typo. It would be better to use a more descriptive name, like 'Props'.